ui/core/components/individual_sim_ui/importers/individual_addon_importer.tsx

```diff
@@ -134,3 +139,12 @@ function glyphToID(glyph: string | JsonObject, db: Database, glyphsConfig: Recor
 	// Cata version exports glyph information in a table that includes the name and the glyph spell ID.
 	return db.glyphSpellToItemId(glyph.spellID as number);
 }
+
+function getWSEVersion(): Promise<string> {
```

Comment:
make sure to catch this and return null, and if it's null just don't show the popup